aws-ec2/README.md

```diff
@@ -72,6 +73,68 @@ For more information about the filters, see <a target="_blank" href="http://docs
 
 For more information about the metrics emitted by Amazon EC2, visit the service's homepage at <a target="_blank" href="https://aws.amazon.com/ec2/">https://aws.amazon.com/ec2/</a>.
 
+### RECOMMENDED Metrics
+
+The following are a subset of available metrics; these are recommended by Amazon for collection.
+
+
+_Standard EC2 Metrics_
+
+CPUUtilization
```

Comment:
This list is pretty plain... I would suggest we present these as a table

Reply:
Done.